packages/hub/services/push-notifications/firebase.ts

```diff
@@ -0,0 +1,24 @@
+import admin from 'firebase-admin';
+
+interface FCMPayload {
+  message: {
+    notification: {
+      body: string;
+      title: string;
+      data: any; // TODO: Define deep link
+    };
+  };
+  token: string;
+}
+
+export default class FirebasePushNotifications {
+  async send(payload: FCMPayload) {
+    await admin.messaging().send(payload);
```

Comment:
Error handling is missing here. But I think we can add this later when we start testing and see the possible edge cases